deployment/legacy-migration/maintenance-worker/migration/neo4j/comments/comments.cql

```diff
@@ -55,6 +55,8 @@ ON CREATE SET
 c.content        = comment.content,
 c.contentExcerpt = comment.contentExcerpt,
 c.deleted        = comment.deleted,
+c.createdAt      = comment.createdAt.`$date`,
+c.updatedAt      = comment.updatedAt.`$date`,
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-13T14:31:40Z" title="Friday, September 13th 2019, 4:31:40 pm +02:00">Sep 13, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -55,6 +55,8 @@ ON CREATE SET
 c.content        = comment.content,
 c.contentExcerpt = comment.contentExcerpt,
 c.deleted        = comment.deleted,
+c.createdAt      = comment.createdAt.`$date`,
+c.updatedAt      = comment.updatedAt.`$date`,
```


I suppose you have tested it.
Therefore I approve. 💫

But where is this `$date` coming from?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-13T18:30:11Z" title="Friday, September 13th 2019, 8:30:11 pm +02:00">Sep 13, 2019</time>_

---


the date would be coming from what is in the database in the `alpha`, I would guess... this is the same way we are setting the date for `Post`s